game-core/src/test/java/games/strategy/engine/vault/VaultTest.java

```diff
@@ -71,6 +72,8 @@ void testLocal() throws Exception {
    * Passes when run individually. Fails when run as part of a suite that consists of multiple
    * server/vault tests.
    */
+  @Disabled
```

Comment:
Noticed these tests that were very old-school disabled, instead of having had '@Ignore' put on them, they had the '@Test' removed. The '@Disabled' is added so we see the disabled test and have a better chance to address the non-determinism or just drop the test code altogether.